sdk/resourcemanager/automanage/armautomanage/README.md

```diff
@@ -38,7 +48,9 @@ For more information on authentication, please see the documentation for `aziden
 Azure Automanage modules consist of one or more clients.  A client groups a set of related APIs, providing access to its functionality within the specified subscription.  Create one or more clients to access the APIs you require using your credential.
 
 ```go
-client, err := armautomanage.NewReportsClient(<subscription ID>, cred, nil)
+reportsClient, err := armautomanage.NewReportsClient(<subscription ID>, cred, nil)
+configProfilesClient, err := armautomanage.NewConfigurationProfilesClient("<sub ID>", cred, nil)
+assignmentClient, err := armautomanage.NewConfigurationProfileAssignmentsClient("<sub ID>", cred, nil)
```

Comment:
Could you please consolidate the placeholder for `subscriptionID`?

Reply:
Done